d120/templates/djangocms_bootstrap4/link/default/link.html

```diff
@@ -0,0 +1,26 @@
+{% load cms_tags %}{% spaceless %}
+  {# this needs to be in one line for rendering purpose #}
+<a
+        href="{{ link }}"
+        {% if instance.target %}
+        target="{{ instance.target }}"
+        {% endif %}
+
+        {% if instance.attributes %}
+          {{ instance.attributes_str }}
+        {% endif %}
+>
+  {% if instance.icon_left %}
+    <i class="{{ instance.icon_left }}"></i>
+  {% endif %}
+  {% for plugin in instance.child_plugin_instances %}
+    {% render_plugin plugin %}
+  {% empty %}
+    {{ instance.name }}
+  {% endfor %}
+
+  {% if instance.icon_right %}
+    <i class="{{ instance.icon_right }}"></i>
+  {% endif %}
+</a>
+{% endspaceless %}
```

Comment:
Blank line at end of file missing

Reply:
Das muss glaube ich so, wenn wir keinen Zeilenumbruch und damit Leerzeichen nach Buttons haben wollen

Reply:
OK :-)